examples/acados_matlab_octave/test/test_all_examples.m

```diff
@@ -29,105 +29,113 @@
 
 %
 
-clear; clc; close all;
-p = path;  % save current path for loading later
-start_dir = pwd;
+clearvars; clc; close all;
 
-% list the examples you would like to test
-targets = {
-%     '../control_rates/main.m';
-%     '../generic_dyn_disc/disc_dyn_example_ocp.m';
-%     '../generic_external_cost/external_cost_example_ocp.m';
-%     '../getting_started/extensive_example_ocp.m';
-%     '../getting_started/minimal_example_closed_loop.m';
-%     '../getting_started/minimal_example_sim.m';
-%     '../getting_started/simulink_example.m';
-%     '../getting_started/simulink_example_advanced.m';
-%     '../linear_mass_spring_model/example_closed_loop.m';
-%     '../linear_mass_spring_model/example_ocp.m';
-%     '../linear_mass_spring_model/example_sim.m';
-%     '../linear_mpc/example_closed_loop.m';
-%     '../lorentz/example_mhe.m';
-%     '../masses_chain_model/example_closed_loop.m';
-%     '../masses_chain_model/example_ocp.m';
-%     '../pendulum_dae/example_closed_loop.m';  % error while reshaping cost.Vz_0
-%     '../pendulum_dae/example_sim.m';
-%     '../pendulum_on_cart_model/example_closed_loop.m';
-%     '../pendulum_on_cart_model/example_ocp.m';
-%     '../pendulum_on_cart_model/example_ocp_custom_hess.m';
-%     '../pendulum_on_cart_model/example_ocp_param_sens.m';
-%     '../pendulum_on_cart_model/example_ocp_reg.m';
-%     '../pendulum_on_cart_model/example_sim.m';
-%     '../pendulum_on_cart_model/example_solution_sens_closed_loop.m';
-%     '../pendulum_on_cart_model/experiment_dae_formulation.m';
-%     '../race_cars/main.m';
-%     '../simple_dae_model/example_ocp.m';
-%     '../swarming/example_closed_loop.m';
-%     '../swarming/example_ocp.m';
-%     '../swarming/example_sim.m';
-%     '../wind_turbine_nx6/example_closed_loop.m';
-%     '../wind_turbine_nx6/example_ocp.m';
-%     '../wind_turbine_nx6/example_sim.m';
+run_tests();
 
-%     './test_checks.m';
-%     './test_mhe_lorentz.m';
-%     './test_ocp_OSQP.m';
-%     './test_ocp_linear_mass_spring.m';
-%     './test_ocp_pendulum_dae.m';
-%     './test_ocp_pendulum_on_cart.m';
-%     './test_ocp_qpdunes.m';
-%     './test_ocp_simple_dae.m';
-%     './test_ocp_wtnx6.m';
-%     './test_sens_adj.m';
-%     './test_sens_forw.m';
-%     './test_sens_hess.m';
-%     './test_sim_dae.m';
-%     './test_target_selector.m'
-};
+function run_tests()
```

Comment:
Overall the diffs are bit harder to review because of the added indent.
Is there are reason to make this a function?
Also in Matlab, file names should correspond to function names.